chart/proxmox-exporter/Chart.yaml

```
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.6
+version: 0.2.7
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "v0.2.6"
+appVersion: "v0.2.7"
```

```suggestion
appVersion: "v0.2.6"
```
The version just happened to collide here but we won't need to release a new version of proxmox-exporter to release a version of the helm chart, so this should stay at v0.2.6

---

Ok, good to know.  I assumed they were supposed to be in lock step :)